File: service/src/main/java/ontology/Algorithm.java
package ontology;

/**
 * Created by esteves on 01.08.15.
 */
public class Algorithm {

    private final String label;
    private final String className;

    public Algorithm(long id, String content) {
        this.id = id;
        this.content = content;
    }

    public long getId() {
        return id;
    }

    public String getContent() {
        return content;
    }

}

File: framework/node/vo/algo/Algorithm.js
/**
 * Created by esteves on 11.07.15.
 */
var Util = require('../../util/mexconstant.js');

// constructor
function Algorithm(ind, className) {
    this._individualName = ind;
    this._className = className;
}
// class methods
Algorithm.prototype.getIndividualName = function() {
    return this._individualName;
};
Algorithm.prototype.getClassName = function() {
    return this._className;
};
Algorithm.prototype.setIndividualName = function(value) {
    this._individualName = value;
};
Algorithm.prototype.setClassName = function(value) {
    this._className = value;
};

// export the class
module.exports = Algorithm;
